src/index.html

```diff
@@ -5,9 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="imac-image"
+        class="product-card__image">
+      <div class="product-card__description">
+       APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="product-card__code">
+        Product code: 195434
+      </div>
+      <div class="product-card__review">
+        <div class="stars">
+          <div class="stars__active"></div>
+          <div class="stars__active"></div>
+          <div class="stars__active"></div>
+          <div class="stars__active"></div>
+          <div class="stars__disabled"></div>
+        </div>
```

Comment:
It would be better to implement this logic using modifiers

<img width="812" alt="Screenshot 2023-07-23 at 12 59 54" src="https://github.com/mate-academy/layout_product-cards/assets/32853034/0c863bc6-ef01-46f9-9281-6e1f2105a89e">

```suggestion
        <div class="stars starts--4">
          <div class="stars"></div>
          <div class="stars"></div>
          <div class="stars"></div>
          <div class="stars"></div>
          <div class="stars"></div>
        </div>
```